easybuild/easyblocks/h/hadoop.py

```diff
@@ -62,6 +70,17 @@ def install_step(self):
         else:
             super(EB_Hadoop, self).install_step()
 
+    def post_install_step(self):
+        """After the install, copy libsnappy into place."""
+        for native_library, lib_path in self.cfg['extra_native_libs']:
+            lib_root = get_software_root(native_library)
+            if not lib_root:
+                self.log.warn("%s not found. Skipping install" % native_library)
+                continue
+            lib_src = os.path.join(lib_root, lib_path)
+            lib_dest = os.path.join(self.installdir, 'lib')
+            shutil.copytree(lib_src, lib_dest)
```

Comment:
wrap this in `try, except OSerror`, and raise `EasyBuildError` in the `except` bit (we actually need a proper function for that in `easybuild.tools.filetools`)